src/pages/login-page.tsx

```diff
@@ -1,15 +1,61 @@
-import React from 'react';
-import {View, Text, StyleSheet} from 'react-native';
+import React, {useState} from 'react';
+import {View, Text, StyleSheet, Alert} from 'react-native';
 import {MainButton} from '../components/main-button';
 import {MainInput} from '../components/main-input';
 
 export const LoginPage: React.FC = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [emailError, setEmailError] = useState('');
+  const [passwordError, setPasswordError] = useState('');
+
+  const validateEmail = (text: string) => {
+    if (!text.trim()) return 'Campo obrigatório';
+    const emailRegex = /^[^\s@]+@[^\s@]+\.com$/;
+    if (!emailRegex.test(text))
+      return 'E-mail inválido (deve terminar com .com)';
```

Comment:
```suggestion
    if (!emailRegex.test(text)) {
      return 'E-mail com formato inválido';
    }
```

Nem todos precisam terminar com .com